apps/service_broadcast/lib/broadcast/stream/broadway.ex

```diff
@@ -9,8 +9,8 @@ defmodule Broadcast.Stream.Broadway do
 
   @app_name get_config_value(:app_name, required: true)
 
-  getter(:broadway_config, required: true)
   getter(:dlq, default: Broadcast.DLQ)
+  getter(:configuration, default: Broadcast.Stream.Broadway.Configuration)
```

Comment:
if this is specific to broadway, i actually prefer the old `broadway_config` getter name to make it clear you're dealing with _just_ broadway's config.